packages/build/src/core/main.js

```diff
@@ -233,9 +234,20 @@ const initAndRunBuild = async function({
   logs,
   timers,
   testOpts,
+  buildbotServerSocket,
 }) {
   const constants = await getConstants({ configPath, buildDir, functionsDistDir, netlifyConfig, siteInfo, mode })
 
+  var buildbotClient
+  if (buildbotServerSocket) {
```

Comment:
This is already checked at the beginning of `startBuildbotClient()`, so can probably be skipped?